k8s/environments/stg/common-overlay/flux-patch.yaml

```diff
@@ -51,13 +51,13 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   annotations:
-    filter.fluxcd.io/nodejs: glob:prod-*
+    filter.fluxcd.io/nodejs: glob:aat-*
   name: pip-frontend
   namespace: pip
 spec:
   values:
     nodejs:
-      image: sdshmctspublic.azurecr.io/pip/frontend:prod-51d4bf6-20210826152156
+      image: sdshmctspublic.azurecr.io/pip/frontend:aat-51d4bf6-20210826152156
```

Comment:
```suggestion
      image: sdshmctspublic.azurecr.io/pip/frontend:prod-51d4bf6-20210826152156
```

Reply:
do I not want to use latest here?

Reply:
no latest is just for initial use so you don't have to bother looking up the image tag, flux will write the actual latest version here, `latest` isn't recommended for actual running